### PR TITLE
Add cloudflare support

### PR DESCRIPTION
**Pull request by nachogarcia**
Add cloudflare support according to https://support.cloudflare.com/hc/en-us/articles/229373388-Cache-Content-by-Device-Type-Mobile-Tablet-Desktop-

### CODE REVIEW

**Comment by dotneet** [comment]
Hi, @nachogarcia 
Thank you for the PR!

According to the reference, headers property has lower cased keys.
Does your code work correctly on your environment?
https://nuxtjs.org/api/context/#-code-req-code-a-href-https-nodejs-org-api-http-html-http_class_http_incomingmessage-em-http-request-em-a-
https://nodejs.org/api/http.html#http_message_headers

**Comment by nachogarcia** [comment]
You're completely correct. Looks like I have setup my testing environment wrong.

**Comment by dotneet** [comment]
I'll check this and merge if everything looks good.

File: lib/plugin.js [hunk 1]
```diff
@@ -66,8 +66,8 @@ export default async function (ctx, inject) {
       mobile = false
       mobileOrTablet = true
     }
-  } else if (ctx.req.headers['CF-Device-Type']) { // Cloudflare
-    switch (ctx.req.headers['CF-Device-Type']) {
+  } else if (ctx.req.headers['cf-device-type']) { // Cloudflare
```
Comment (dotneet) [inline lib/plugin.js]: Check if `ctx.req` is defined.

**Comment by nachogarcia** [comment]
Let me know if more changes are required.

**Comment by dotneet** [comment]
No need more changes.  I'll merge this after check.

**Comment by dotneet** [comment]
Thanks!

**Comment by nachogarcia** [comment]
No problem :)
Will you release it soon?

**Comment by dotneet** [comment]
@nachogarcia 
I've released just now!

**Pull request merged**